cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,163 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import { faker } from '@faker-js/faker';
+import { generateNewArticle } from '../support/generateNewArticle';
 
-describe('Article', () => {
-  before(() => {
 
-  });
 
-  beforeEach(() => {
-    cy.task('db:clear');
-  });
+describe('Create Article page', () => {
 
-  it('should be created using New Article form', () => {
+   let user;
+   const errorMsg = 'Oops!';
+   const { title, description, body, tags } = generateNewArticle();
 
-  });
+   beforeEach(() => {
+      cy.task('db:clear');
+      cy.task('generateUser').then(generateUser => {
+         user = generateUser;
+      });
+      cy.register().then(user => {
```

Comment:
Replace 'cy.login(user).then(() => user);' with 'cy.login(user);'. The 'then' statement is not necessary here.